changelogs/unreleased/gh-4348-transactional-ddl.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
+
+* All the DDL functions from `box.schema` module are wrapped into transaction
+  to avoid database inconsistency on failed operation (gh-4348).
```

Comment:
```suggestion
* All DDL functions from the `box.schema` module are now wrapped into a transaction
  to avoid database inconsistency on failed operations (gh-4348).
```